apps/web/layout/HomeLayout.tsx

```diff
@@ -13,8 +13,13 @@ export const HomeLayout = ({ children }: Child) => {
     const dispatch = useDispatch();
 
     useEffect(() => {
+        const unProtectedRoutes = ['/events'];
```

Comment:
these routes are static right ? put these outside the component so that it wont rerender on state change